routes/api.php

```diff
@@ -66,6 +66,9 @@
 Route::post('/reset-password', [UserController::class, 'resetPassword']);
 Route::post('/send-verification-email', [UserController::class, 'sendVerificationEmail']);
 Route::post('/verify-email', [UserController::class, 'verifyEmail']);
+Route::post('/enable-2fa', [UserController::class, 'enable2FA'])->middleware('auth:sanctum');
+Route::post('/generate-otp', [UserController::class, 'generateOtp'])->middleware('auth:sanctum');
```

Comment:
you can delete the generate-otp and the verify-otp enpints since you call them bith in the login endpoint